lunr/query.py

```diff
@@ -81,18 +85,17 @@ def term(self, term, **kwargs):
             query.term(lunr.Tokenizer("foo bar"))
 
         Args:
-            term (Token or iterable): Token or iterable of tokens to add.
+            term (str, Token or iterable): Token or iterable of tokens to add.
             kwargs (dict): Additional properties to add to the Clause.
         """
-        if isinstance(term, (list, tuple)):
+        if isinstance(term, str) or isinstance(term, Token):
```

Comment:
Inverted the condition here because Iterable is not just list and tuple (also, `str` is Iterable, which is definitely *not* what you want!)